### ISSUE TITLE
toga_gtk.widgets.internal moduel not found

### ISSUE THREAD

**Issue by fadyadal** [beeware/toga#1111]
**Describe the bug**
toga-demo doesn't start. When run, the following error occurs.
![image](https://user-images.githubusercontent.com/24705058/96589679-636f9580-12e5-11eb-9e72-dde4ad6d5d89.png)

**To Reproduce**
1. I am on manjaro so running 
```sh
$ sudo pacman -Syu git pkgconf cairo python-cairo pango gobject-introspection gobject-introspection-runtime python-gobject webkit2gtk
```
2. Installing toga-demo
```sh
$ sudo pip -H install --pre toga-demo
```
3. Running toga-demo
```sh
$ toga-demo
```

**Expected behavior**
Toga Demo starts

**Screenshots**
See above.

**Environment:**
 - Operating System: Manjaro
 - Python version: 3.8
 - Software versions:
   - Briefcase: N/a
   - Toga: 0.3.0.dev24
   - ...

**Additional context**

**Comment by rajatkulshreshtha** [comment]
I am facing same problem on linux mint 19
however on Manjaro 20.1.1 I am facing issues too,
Can anyone suggest me which OS should I use to get reliable output?

**Comment by freakboy3742** [comment]
My apologies - it looks like this was a packaging error in the dev25 release, caused by a missing `__init__.py` file. I've corrected the problem in #1114; I'll release 0.3.0dev25 shortly to ensure there's a working GTK install available.
